### ISSUE TITLE
macOS Support

### ISSUE THREAD

**Issue by sclsj** [micsthepick/JSFX-Signal-Subtraction#1]
Can I make this work on macOS?

**Comment by micsthepick** [comment]
Yes, I have made JSFX plugins work with MacOS, there are a few options, you can install reaper, or use wine to run your audio application (I've done this, with a 64 bit limited system with some tricks), one last option is ysfx, but I have not tested that option. If you can do some research into each of these options and if you get stuck, let me know what you've done, and I can try to help

**Comment by micsthepick** [comment]
I just tested YSFX with the 64 bit version of Audacity 3.1.3, and macOS 11.6.5, all went well, except when I didn't use 44.1KHz as the samplerate, in fact it removes a problem with ReaJS where the parameters would not dynamically update
